staging/src/k8s.io/controller-manager/pkg/features/kube_features.go

```diff
@@ -48,13 +34,12 @@ const (
 	StableLoadBalancerNodeSet featuregate.Feature = "StableLoadBalancerNodeSet"
 )
 
-func SetupCurrentKubernetesSpecificFeatureGates(featuregates featuregate.MutableFeatureGate) error {
-	return featuregates.Add(cloudPublicFeatureGates)
+func init() {
+	runtime.Must(utilfeature.DefaultMutableFeatureGate.Add(cloudPublicFeatureGates))
```

Comment:
> Why is it in this PR?

As it was: `SetupCurrentKubernetesSpecificFeatureGates` was only called from the init function in  [`package register`](https://github.com/kubernetes/kubernetes/blob/b3d8ac8496a23d65a907f9333d906bcd5463764e/staging/src/k8s.io/controller-manager/pkg/features/register/register.go), forcing any test file to reference that package as to test code depending on the feature gate. Right now, this follows the general convention across the code base and forces test files to just call [`package features`](https://github.com/kubernetes/kubernetes/blob/1984eeff3bc39fd0595099a6ce2d5b686ee064a9/staging/src/k8s.io/controller-manager/pkg/features/kube_features.go) - which needs to happen in any case when accessing the variable `StableLoadBalancerNodeSet`. 

This was never picked up on before, because the feature gate `IPv6DualStack` was never tested from the controller-manager.